new-backend/app/modules/langgraph_nodes/fact_check.py

```
@@ -0,0 +1,30 @@
+    return []
+
+
+def run_fact_check(state):
+    try:
+        text = state.get("cleaned_text")
+        keywords = state["keywords"]
+
+        if not text:
+            raise ValueError("Missing or empty 'cleaned_text' in state")
+        elif not keywords:
+            raise ValueError("Missing or empty 'keywords' in state")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "raise", remove the leading "el" from "elif"

(R1720)